<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetClipping</class>
 <widget class="QWidget" name="WidgetClipping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clipping</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPoint">
       <property name="text">
        <string>Point</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="coordinateEditPointX">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000</double>
       </property>
       <property name="maximum">
        <double>10000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="coordinateEditPointY">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000</double>
       </property>
       <property name="maximum">
        <double>10000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="coordinateEditPointZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000</double>
       </property>
       <property name="maximum">
        <double>10000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNormal">
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="coordinateEditNormalX">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000</double>
       </property>
       <property name="maximum">
        <double>10000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="coordinateEditNormalY">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000</double>
       </property>
       <property name="maximum">
        <double>10000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="coordinateEditNormalZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000</double>
       </property>
       <property name="maximum">
        <double>10000</double>
       </property>
       <property name="value">
        <double>1</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkIsClipping">
       <property name="text">
        <string>Clipping</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkIsCapping">
       <property name="text">
        <string>Capping</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
